src/aave-v3/SupplyVault.sol

```diff
@@ -29,8 +29,8 @@ contract SupplyVault is ISupplyVault, SupplyVaultBase {
     event Accrued(
         address indexed rewardToken,
         address indexed user,
-        uint128 rewardsIndex,
-        uint128 accruedRewards
```

Comment:
I think we can keep uint128 here, why changing it?

Reply:
It saves some gas